enos/utils/enostask.py

```diff
@@ -81,6 +82,15 @@ def decorated(*args, **kwargs):
 
             # Constructs the environment
             kwargs['env'] = make_env(kwargs['--env'])
+
+            # Check if the environment has been loaded for the commands
+            # requiring it
+            if fn.__name__ not in ['up', 'deploy', 'info'] and \
```

Comment:
Yes, checking if the linked env is one thing.
But I think the original idea was to validate that some specific keys were present/not None in the env for a given task. That's why I proposed a specific decorator to separate the concerns. It looks weird to me to reference the phases (`up`, `deploy`, ...) in this file.